src/Tasks/GetSDKReferenceFiles.cs

```diff
@@ -1246,63 +1218,79 @@ public SdkReferenceInfo(string fusionName, string imageRuntime, bool isWinMD, bo
         /// Structure that contains the on disk representation of the SDK in memory.
         /// </summary>
         /// <remarks>This is a serialization format. Do not change member naming.</remarks>
-        [Serializable]
-        private class SDKInfo
+        internal class SDKInfo : ITranslatable
         {
-            // Current version for serialization. This should be changed when breaking changes
-            // are made to this class.
-            private const byte CurrentSerializationVersion = 1;
+            private Dictionary<string, SdkReferenceInfo> pathToReferenceMetadata;
+            private Dictionary<string, List<string>> directoryToFileList;
+            private int hash;
```

Comment:
Our/runtime's standard naming convention would make these:

```suggestion
            private Dictionary<string, SdkReferenceInfo> _pathToReferenceMetadata;
            private Dictionary<string, List<string>> _directoryToFileList;
            private int _hash;
```